flink-table-store-hive/pom.xml

```diff
@@ -83,7 +83,50 @@ under the License.
                 <hive.main.version>3</hive.main.version>
                 <hive.version>3.1.2</hive.version>
             </properties>
+            <dependencies>
+                <!-- For Hive 3.1.2, package org.apache.hadoop.hive.metastore.api relies on this dependency-->
+                <dependency>
+                    <groupId>org.apache.hive</groupId>
+                    <artifactId>hive-standalone-metastore</artifactId>
+                    <version>${hive.version}</version>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>log4j</groupId>
+                            <artifactId>log4j</artifactId>
+                        </exclusion>
+                        <exclusion>
+                            <groupId>org.slf4j</groupId>
+                            <artifactId>slf4j-log4j12</artifactId>
+                        </exclusion>
+                </exclusions>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
I don't think we should shade hive in parent module.
We should shade in `flink-table-store-hive-catalog`.